### ISSUE TITLE
 How can I get Included data?

### ISSUE THREAD

**Issue by mirmousaviii** [kamikat/moshi-jsonapi#46]
I use retrofit and I need get data from included.
response:

```
{
	"data": {
		"type": "aaa",
		"id": 1,
		"attributes": {
			"username": "user1",
			"created_at": "2017-01-16T15:12:06+00:00",
			"updated_at": "2017-01-16T15:12:06+00:00"
		},
		"relationships": {
			"rr1": {
				"data": {
					"type": "ccc",
					"id": 2
				}
			}
		}
	},
	"included": [
		{
			"type": "ccc",
			"id": 2,
			"attributes": {
				"title": "title one",
				"description": "description one"
			},
			"relationships": {
				"rr22": {
					"data": {
						"type": "ddd",
						"id": 2
					}
				}
			}
		}
	]
}
```
I can get Id and Type with `response.body().getDocument().included.get(0).getId()`, But How can I get attributes from included? for example title and description

Thank you.

**Comment by mirmousaviii** [comment]
@kamikat Can you help me?

**Comment by kamikat** [comment]
`response.body().getDocument().included.get(0)` gets just an instance of class with `@JsonApi(type = "ccc")` in your example. And you can access its fields (attributes/relationships) after cast to corresponding class.

BTW, `Document.included` is made unaccessible in 3.0. Could you please provide more detail about your practice?

**Comment by mirmousaviii** [comment]
It's resolved after cast to corresponding class.
@kamikat Thank you for every help.
Thank you.
